galcv/__init__.py

```diff
@@ -371,3 +378,70 @@ def log10Eps(area, Psi, gamma, b):
         The parameters of the varepsiloncv fit
     '''
     return Psi * area**gamma + b
+
+
+def absToApp(Mabs='ER', z='ER'):
```

Comment:
These new methods would be an excellent time to start breaking this file apart into more cohesive and distinct files/modules.

Reply:
That's true. That will probably take a little work to implement and make sure I'm doing it the correct way. Lets leave that to a future pull request. I'll add it as an issue.